binder/src/main/java/io/grpc/binder/internal/BinderTransport.java

```diff
@@ -637,6 +637,12 @@ public synchronized ClientStream newStream(
         ClientStreamTracer[] tracers) {
       if (isShutdown()) {
```

Comment:
Move this condition inside the next one? (isShutdown() is a subset of != READY).